cwl/src/main/scala/cwl/ExpressionEvaluator.scala

```diff
@@ -24,23 +24,31 @@ object ExpressionEvaluator {
   type ECMAScriptFunction = String Refined MatchesRegex[ECMAScriptFunctionWitness.T]
   type MatchesECMAFunction = MatchesRegex[ECMAScriptFunctionWitness.T]
 
-  def evalExpression(expression: ECMAScriptExpression)(parameterContext: ParameterContext): ErrorOr[WomValue] = {
+  def evalExpression(expression: ECMAScriptExpression, parameterContext: ParameterContext, expressionLib: ExpressionLib): ErrorOr[WomValue] = {
     expression.value match {
-      case ECMAScriptExpressionRegex(script) => eval(script, parameterContext)
+      case ECMAScriptExpressionRegex(script) =>
+        eval(expressionFromParts(expressionLib, script), parameterContext)
       case unmatched =>
         s"Expression '$unmatched' was unable to be matched to regex '${ECMAScriptExpressionWitness.value}'".invalidNel
     }
   }
 
-  def evalFunction(function: ECMAScriptFunction)(parameterContext: ParameterContext): ErrorOr[WomValue] = {
+  def expressionFromParts(lib: ExpressionLib, script: String) = {
+    val expressionScript: String = lib.mkString(";")
+    s"$expressionScript;$script"
```

Comment:
Am I correct in saying in almost everything in the PR is wiring to get here and add this before the script ?

Reply:
That is correct!  Quite a ripple effect.

Reply:
I think ultimately we can mitigate the ugliness w/ parameters/dependency injection by using the `Reader` monad, but I didn't feel brave enough to introduce it in this PR.